jpa/subsystem/src/main/java/org/jboss/as/jpa/transaction/TransactionUtil.java

```diff
@@ -45,19 +53,29 @@
 
     private static volatile TransactionSynchronizationRegistry transactionSynchronizationRegistry;
     private static volatile TransactionManager transactionManager;
+    private static volatile TransactionListenerRegistry transactionListenerRegistry;
```

Comment:
All these statics kinda scare me. Why do we need these, as opposed to using services?